predicators/spot_utils/skills/spot_sweep.py

```diff
@@ -0,0 +1,148 @@
+"""Interface for spot sweeping skill."""
+
+import numpy as np
+from bosdyn.client import math_helpers
+from bosdyn.client.sdk import Robot
+
+from predicators.spot_utils.skills.spot_hand_move import \
+    move_hand_to_relative_pose
+
+
+def sweep(robot: Robot, sweep_start_pose: math_helpers.SE3Pose,
+          sweep_yaw: float, sweep_distance: float) -> None:
+    """Sweep in the xy plane, starting at the start pose and then moving in the
+    yaw direction for the given distance."""
+    # First, move the hand to the start pose.
+    move_hand_to_relative_pose(robot, sweep_start_pose)
+    # Calculate the end pose.
+    relative_hand_move = math_helpers.SE3Pose(
+        x=sweep_distance * np.cos(sweep_yaw),
+        y=sweep_distance * np.sin(sweep_yaw),
+        z=0,
+        rot=math_helpers.Quat())
+    sweep_end_pose = relative_hand_move * sweep_start_pose
+    # Move the hand to the end pose.
+    move_hand_to_relative_pose(robot, sweep_end_pose)
```

Comment:
Wait - given this implementation, the sweep will happen in a straight line right? In that case, what's the point of this "yaw" parameter (I was imagining this would create some kind of circular sweep, but then the implementation doesn't make sense)?